TweekApiSmokeTestDockerfile

```diff
@@ -1,6 +1,8 @@
 FROM microsoft/aspnetcore-build:2.0.0
 
-RUN wget https://raw.githubusercontent.com/vishnubob/wait-for-it/8ed92e8cab83cfed76ff012ed4a36cef74b28096/wait-for-it.sh && \
+RUN apt-get update && \
+    apt-get install -y jq && \
```

Comment:
there are some best practices like `--no-recomendment` (not remember the syntax) and clear cache to make image smaller, not sure if it important here